README.md

```diff
@@ -156,6 +185,79 @@ CREATE TABLE compound (
 
 ## Clustering keys and Sorting
 
+Cassandra stores data on each node according to the hashed TOKEN value of the partition key in the range that the node is responsible for.
+Since hashed TOKEN values are generally random, `find` with `limit: 10` filter will return apparentrly random 10 (or less) rows.
+The Cassandra connector supports on-disk sorting by setting `clustering key` as ASCending or DESCending at table creation time.
+`order` filter is ignored.  Since sorting is done on node by node basis, the returned result is property sorted only when the partition key is specified.
+
+For example, in case you want to find the most recently added row, create a table with time-based column as a clustering key with DESC property.
+Then, use `find` with `limit: 1` or `findOne`.
+
+Concreate example is as follows assuming all the rows fall in the same partition range:
```

Comment:
Can we make the table/column name more meaningful for real-world? For example, a customer table with region/email/firstName/lastName etc?